SFML/SFML
# Update to Catch2 v2.13.7

## Description

Update to the latest release of Catch2, version 2.13.7. Downloaded from [here](https://github.com/catchorg/Catch2/releases/tag/v2.13.7). The previous version was Catch v1.12.2 from May 2018 so this is a fairly big leap into the present.

This PR is related to the issue #1897 

## Tasks


## How to test this PR?

Build tests before and after with macOS to observe the compilation error being fixed.
